jest.config.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
     ],
   ],
   testResultsProcessor: 'jest-junit',
+  testRunner: "jest-circus/runner"
```

Comment:
`jest-circus` is the default, so I don't think we need to declare it.

https://jestjs.io/docs/configuration#testrunner-string